src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
-body {
+@import 'variables';
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+@each $day, $index in $days {
+  $index: $index + 6;
+  .calendar--start-day-#{$day} > :first-child {
+    margin-left: calc((#{$calendar-day-size} + #{$calendar-gap}) * #{$index});
+  }
```

Comment:
The loop variable `$index` is being reassigned on line 20 which can lead to confusion and unexpected behavior. It's generally a bad practice to modify loop variables. Consider using a different variable name for the calculated value.